css/ampshared.css

```diff
@@ -229,7 +231,9 @@ i-amphtml-sizer {
 }
 
 .i-amphtml-notbuilt,
-[layout]:not(.i-amphtml-element)
+[layout]:not(.i-amphtml-element),
+[width][height][sizes]:not([layout]):not(.i-amphtml-element),
+[width][height][heights]:not([layout]):not(.i-amphtml-element)
```

Comment:
Please add a comment that this `[width][height][sizes]` and `[width][height][heights]` are "implicit responsive" here and below. Follow-up PR is cool too to avoid the bundle size reapproval.